base/docs/basedocs.jl

```diff
@@ -675,6 +675,14 @@ generation specialization for that field.
 ANY
 
 """
+    Core.TypeofBottom
+    Type{Union{}}
```

Comment:
Yes, it's currently not true. Jeff and Jameson are having an ongoing debate about whether this should be true or not. Consider it aspirational.